docs/DeveloperGuide.md

```diff
@@ -116,6 +116,7 @@ It is made up of three major components:
 1. `AddressBook` which manages the contacts in the form of `Person` objects.
 1. `Calendar` which manages the events in the form of `Event` objects.
 1. `TagTree` which manages `tag` to `tag` relations.
+1. `Reminder` which manages `reminder` to `event` relations.
```

Comment:
By the way, I believe this should be 'RemindersImpl` as the ones are above are all the collections of `Person`s, `Tag`s, etc.

Reply:
changed and resolved